.travis.yml

```diff
@@ -23,13 +27,14 @@ script:
   - ipython profile create qiita_general --parallel
   - qiita_env start_cluster qiita_general
   - qiita_env make --no-load-ontologies
-  - qiita_test_install
-  - nosetests --with-doctest --with-coverage
-  - flake8 qiita_* setup.py scripts/qiita scripts/qiita_env scripts/qiita_test_install
+  - if [ ${TEST_ADD_STUDIES} == "True" ]; then test_data_studies/commands.sh ; fi
+  - if [ ${TEST_ADD_STUDIES} == "False" ]; then qiita_test_install ; fi
```

Comment:
If everything was in the same `if` all but the last return codes would be hidden (this already caused a false green build when one command was `cd ; failed command; cd ..`).

Agree it's not optimal, and if travis.yml gets more complex we could add a makefile like in skbio.